packages/aws-cdk-lib/aws-apigatewayv2/lib/parameter-mapping.ts

```diff
@@ -16,6 +16,18 @@ export class MappingValue implements IMappingValue {
    * Creates an empty mapping value.
    */
   public static readonly NONE = new MappingValue('');
+  /**
+   * [PLACEHOLDER COMMENT: TO BE FILLED OUT]
+   */
+  public static readonly JWT = new MappingValue('JWT');
+  /**
+   * [PLACEHOLDER COMMENT: TO BE FILLED OUT]
+   */
+  public static readonly CUSTOM = new MappingValue('CUSTOM');
+  /**
+   * [PLACEHOLDER COMMENT: TO BE FILLED OUT]
+   */
+  public static readonly AWS_IAM = new MappingValue('AWS_IAM');
```

Comment:
Given the format of some of the other parameters in this class, I'm not sure these values are correct. We should dive deeper in this case and make sure these are being mapped correctly.